services-js/public-notices/src/model/Notice.ts

```diff
@@ -1,10 +1,10 @@
 /**
  * Type of a public notice from the Boston.gov public notices v1 API:
  *
- * https://www.boston.gov/api/v1/public-notices
+ * https://www.boston.gov/api/v2/public-notices
  */
 export default interface Notice {
-  is_testimony: '';
+  is_testimony: '0' | '1';
```

Comment:
Could this become a boolean value instead?

Reply:
It’s not straightforward to, due to the way the data is getting pulled out of Drupal.

Reply:
👍 figured there must have been a good reason, but wanted to ask!